apps/signal/56-forms-and-signal/src/app/checkout.component.ts

```diff
@@ -15,6 +15,7 @@ import { products } from './products';
     <h2 class="mb-1 w-full bg-gray-400 p-2 text-white">Checkout</h2>
     <button
       routerLink="/order"
+      [queryParams]="{ quantity: quantity() }"
```

Comment:
why do you need this one?

Reply:
am passing the quantity data as part of the url in order to retrieve it on the order.component.ts  when we use the ActivatedRoute